src/en/help-azure-advanced.md

```diff
@@ -0,0 +1,172 @@
+Title: Using Microsoft Azure with Juju - advanced
+
+# Using Microsoft Azure with Juju - advanced
+
+This page is dedicated to more advanced topics related to using Microsoft Azure
+with Juju. The main page is [here][clouds-azure].
+
+## Manually adding Azure credentials
+
+The manual option is useful if Juju fails to automatically gather your
+credentials, or if you want to automate the process.
+
+### Gathering values
+
+We will need values for the following bits of information:
+
+ - application-id
+ - subscription-id
+ - application-password
+ - application-name
+ - tenant-id
+
+In the sections below, we will assign each of these a variable name.  When you
+enter them into the command, replace the variable name we give with the actual
+ID that corresponds to the variable.
+
+!!! Important:
+    This process requires the Azure CLI tool to be installed and a successful
+    login with it. See [Install the CLI tool][clouds-azure-cli-install] and
+    [Log in to Azure][clouds-azure-cli-login] respectively.
+
+#### `subscription-id`
+
+List your account. Note the subscription ID, the **SUB_ID**.
+
+```bash
+az account list
+```
+
+**SUB_ID** will appear on the line labelled **id**:
+
+```yaml
+[
+  {
+    "cloudName": "AzureCloud",
+    "id": "f717c8c1-8e5e-4d38-be7f-ed1e1c879e18",
+    "isDefault": true,
+    "name": "Pay-As-You-Go",
+    "state": "Enabled",
+    "tenantId": "0fb95fd9-f42f-4c78-94c9-e3d01c2bc5af",
+    "user": {
+      "name": "javierlarin72@gmail.com",
+      "type": "user"
+    }
+  }
+]
+```
+
+In our sample, **SUB_ID** is the second line line, so:
+
+```bash
+SUB_ID=f717c8c1-8e5e-4d38-be7f-ed1e1c879e18
+```
+
+#### `application-password` and `application name`
+
+Create a password for the application to use. You will also need to come up
+with an arbitrary application name (typically an internet domain). In our
+example:
+
+```bash
+APP_PASSWORD=some_password
+APP_NAME=ubuntu.example.com
+```
+
+Now create an Active Directory (Kerberos) server principal and grant the
+required resource permissions by assigning a role of ***Owner***:
```

Comment:
It's not exactly clear to me what's being assigned the *Owner* role.

Reply:
I agree this sounds convoluted. I think I grabbed this from the Microsoft site when I first wrote it. I'll dig a bit.